packages/toolboxes/cosmos/src/util.ts

```diff
@@ -50,6 +62,28 @@ export const getDenom = (symbol: string, isThorchain = false) => {
   }
 };
 
+export const getDenomWithChain = ({ symbol, chain }: AssetValue) => {
+  if (chain === Chain.Maya) {
+    return (symbol.toUpperCase() !== "CACAO" ? symbol : `${Chain.Maya}.${symbol}`).toUpperCase();
+  }
+
+  if (chain === Chain.THORChain) {
+    return (
+      symbol.toUpperCase() !== "RUNE" ? symbol : `${Chain.THORChain}.${symbol}`
+    ).toUpperCase();
+  }
+
+  if (chain === Chain.Kujira) {
+    return (symbol.toUpperCase() !== "KUJI" ? symbol : `${Chain.Kujira}.${symbol}`).toUpperCase();
```

Comment:
should return 'ukuji' imo

Reply:
ah I will have to read through the cosmos stuff again - I dont think we need this method for other chains but MAYA + THOR